.storybook/preview.js

```diff
@@ -23,6 +23,7 @@ const preview = {
           ["Theme", "Fonts"],
           "Examples",
           "Components",
+          "Layercake",
```

Comment:
Ok just one tiny last suggestion: to have the "Intro" page listed at the top of the Layercake section, just add one additional line below "Layercake" here:
```
...
"Layercake",
 ["Intro"],
...
```

Which should result in this ordering:

<img width="373" alt="Screenshot 2024-08-28 at 12 59 39 PM" src="https://github.com/user-attachments/assets/4b50d16f-578c-422c-aa94-ff2ddde02341">